modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,95 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-0_{context}"]
+= Release notes for {gitops-title} 1.12.0
+
+{gitops-title} 1.12.0 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-12.0_{context}"]
+== Errata updates
+
+TBD
```

Comment:
@reginapizza - Do we need to document any Errata entries for this release? Keeping this section until you confirm. If there are no entries, I will delete this section.

Reply:
sorry, I'm not sure what you mean by errata entries?

Reply:
Hi @Dhruv-Soni11, this advisory has now been updated from a RHEA (Red Hat Enhancement Advisory) to a RHSA (Red Hat Security Advisory), so it will need that section now. Please coordinate with @Srivaralakshmi to understand how to get the info for that section as I'm a little unsure about that bit.